api/lisk.js

```diff
@@ -169,18 +166,39 @@ module.exports = function (app) {
             parallel.authenticateAddress
         ], function (error, values) {
             if (error) {
-                return res.json({ success : false, error : error });
+                return res.json({ success: false, error: error });
             } else {
                 async.series([
                     series.validateCaptcha,
                     series.cacheIP,
                     series.cacheAddress,
-                    series.sendTransaction,
+                    series.getAccount,
                 ], function (error, results) {
                     if (error) {
-                        return res.json({ success : false, error : error });
+                        return res.json({ success: false, error: error });
                     } else {
-                        return res.json({ success : true, txId : 'unknown' });
+                        const apiClient = getApiClient(app);
+                        const account = results[4];
+                        const amount = app.locals.amountToSend * req.fixedPoint;
+                        const networkIdentifier = cryptography.getNetworkIdentifier(app.locals.nethash, 'Lisk');
+                        const transaction = transactions.transfer(
+                            {
+                                networkIdentifier,
+                                recipientId: address,
+                                amount: String(amount),
+                                passphrase: app.locals.passphrase,
+                                nonce: account.nonce,
+                                fee: MINIMUM_FEE,
```

Comment:
So fee is always the minimum? Does it make sense to involve the estimates from Lisk Service here?

Reply:
i think for now, minimum_fee should be good for now, but maybe we should consider in the v5

Reply:
yea.. for now we can keep minimum fee @MichalTuleja also its only token transfer.